File: responses/3JJAY9-8X9/reverse_digits.js
// you can write to stdout for debugging purposes, e.g.
// console.log('this is a debug message');

function solution(x) {
    var stringX=x.toString();
    var solution="";
    var size=stringX.length;
    for(var i=0;i<size;i++){
        solution+=stringX[size-i-1];
    }
    var tmp = parseInt(solution);
    if(stringX[0]=='-'){
        return 0-tmp;
    } 
    else{
        return tmp;
    }
}

File: responses/ZUJBHC-KBZ/reverse_digits.py
# you can write to stdout for debugging purposes, e.g.
# print("this is a debug message")

def solution(x):
    temp = 0
    for i in range(x):
        if i%3==0 or i%5==0 and i not in [3,5]:
            temp += i
    return temp